subprojects/core/src/integTest/groovy/org/gradle/api/tasks/TaskInputPropertiesIntegrationTest.groovy

```diff
@@ -589,8 +589,8 @@ task someTask(type: SomeTask) {
 
     @Unroll
     def "wrong output file type registered via TaskOutputs.#method is not allowed"() {
-        file("input-file.txt").touch()
-        file("input-dir").createDir()
+        file("output-file.txt").touch()
+        file("output-dir").createDir()
```

Comment:
Shall we rename the test to `TaskParametersIntegrationTest`?

Reply:
Done.